setup.rst

```diff
@@ -118,31 +105,132 @@ setup your project with the following commands:
     $ cd my-project/
     $ composer install
 
-You'll probably also need to customize your :ref:`.env <config-dot-env>` and do a
-few other project-specific tasks (e.g. creating database schema). When working
-on a existing Symfony app for the first time, it may be useful to run this
-command which displays information about the app:
+You'll probably also need to customize your :ref:`.env file <config-dot-env>`
+and do a few other project-specific tasks (e.g. creating a database). When
+working on a existing Symfony application for the first time, it may be useful
+to run this command which displays information about the project:
 
 .. code-block:: terminal
 
     $ php bin/console about
 
+.. _symfony-flex:
+
+Installing Packages
+-------------------
+
+A common practice when developing Symfony applications is to install packages
+(Symfony calls them :doc:`bundles </bundles>`) that provide ready-to-use
+features. Packages usually require some setup before using them (editing some
+file to enable the bundle, creating some file to add some initial config, etc.)
+
+Most of the times this setup can be automated and that's why Symfony includes
+`Symfony Flex`_, a tool to simplify the installation/removal of packages in
+Symfony applications. Technically speaking, Symfony Flex is a Composer plugin
+that is installed by default when creating a new Symfony application and which
+**automates the most common tasks of Symfony applications**.
+
+.. tip::
+
+    You can also :doc:`add Symfony Flex to an existing project </setup/flex>`_.
+
+Symfony Flex modifies the behavior of the ``require``, ``update``, and
+``remove`` Composer commands to provide advanced features. Consider the
+following example:
+
+.. code-block:: terminal
+
+    $ cd my-project/
+    $ composer require mailer
```

Comment:
I would use another example as `mailer` is the only edge case right now that is a problem as we have `symfony/mailer` and a `mailer` alias on Swiftmailer.